codegen/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
Never-mind. It seems like at one point you couldn't do this, but it appears to be somewhat common now.